src/views/charts/PnlChart.tsx

```diff
@@ -83,7 +80,7 @@ export const PnlChart = ({
   const chartDotsBackground = useAppSelector(getChartDotBackground);
 
   // Chart data
-  const pnlData = useAppSelector(getSubaccountHistoricalPnl, shallowEqual);
+  const { data: pnlData } = BonsaiHooks.useParentSubaccountHistoricalPnls();
```

Comment:
do we need to append one additional data point for `Date.Now()` and current equity?

Reply:
oh nvm it looks like that is happening in the useMemo below